extension/chrome/popups/select_account.ts

```diff
@@ -56,8 +56,9 @@ View.run(
     };
 
     private actionChooseAcctHandler = async (clickedElement: HTMLElement) => {
+      const useFullScreenSecureCompose = Browser.isThunderbirdMail() ? true : false;
```

Comment:
it can be simplified to `const useFullScreenSecureCompose = Browser.isThunderbirdMail()`

Reply:
will do - thanks!